go-controller/pkg/util/util.go

```diff
@@ -261,3 +261,21 @@ ipLoop:
 
 	return out
 }
+
+func GetLogicalPortName(podNamespace, podName string) string {
```

Comment:
ideally the cosmetic changes to creating the port name should be a separate commit in this PR. Will make backporting easier.